xformers/components/attention/nystrom.py

```diff
@@ -43,11 +43,13 @@ class NystromSelfAttentionConfig(AttentionConfig):
 
     num_heads: int
     num_landmarks: Optional[int]
-    landmark_pooling: Optional[nn.Module]
+    # should be nn.Module, OmegaConf does not support the typing
+    landmark_pooling: Optional[Any]
```

Comment:
same here, this basically removes all type checking so that OmegaConf can work, it's a big price to pay I think. Coud this be fixed upstream ?

Reply:
OmegaConf does not support pytorch typing and I don't think we are planning on supporting it any time soon. The reason being Hydra/OmegaConf being a  low level  python lib and we do not want to add dependency on pytorch. Pls check #93 in which I've made this optional :)